ambry-clustermap/src/main/java/com/github/ambry/clustermap/DatacenterInitializer.java

```diff
@@ -158,7 +158,10 @@ private void onInitializationFailure(Exception e) {
    * @throws Exception if something went wrong during startup
    */
   private DcInfo initializeHelixDatacenter() throws Exception {
-    String zkConnectStr = dcZkInfo.getZkConnectStr();
+    // For now, the first ZK endpoint (if there are more than one endpoints) will be adopted by default for initialization.
+    // If we really need multiple HelixClusterManagers on same node in the future, we can use separate zk connect str configs
+    // for HelixClusterManager and HelixParticipant.
```

Comment:
This comment is confusing to me. DatacenterInitializer is not part of the ClusterParticipant. It's used by HelixClusterManager to fetch the instance config and fill the clustermap. Why do we need to support multiple of them?

Reply:
True, we don't need support multiple HelixClusterManager(which is also spectator in the context of Helix). I added the comment to clarify that we have to pick one (and only one) ZK endpoint to connect, as `clusterMapDcsZkConnectStrings` contains several ZK endpoints in each dc to support multiple participants. I will remove the comment and make it clear.